libr/anal/cc.c

```diff
@@ -15,6 +15,30 @@ R_API void r_anal_cc_del(RAnal *anal, const char *name) {
 	}
 }
 
+R_API char *r_anal_cc_from_regprofile(RAnal *anal) {
```

Comment:
Why not this:
```suggestion
R_API char *r_anal_cc_from_regprofile(RReg *reg) {
```

Reply:
No. This will break the naming rules of public APIs. this is under RAnal, so it must use such type as 1st argument. if we move this into reg/profile.c then that would be fine to use RReg. but it's not there right now